gtsfm/evaluation/metrics.py

```diff
@@ -0,0 +1,396 @@
+"""Classes to store metrics computed in different GTSfM modules.
+
+The GtsfmMetric class stores a single metric, and the GtsfmMetricsGroup stores a list of metrics.
+These classes are used to compute statistics on the metrics (min, max, etc), 
+save them to JSON, parse metrics from JSON, and plot them.
+
+Authors: Akshay Krishnan
+"""
+
+from __future__ import annotations
+
+import json
+from enum import Enum
+from typing import Any, Dict, List, Union
+
+import numpy as np
+
+import gtsfm.utils.io as io
+import gtsfm.utils.logger as logger_utils
+
+# Keys to access data and summary in the dictionary representation of metrics.
+DATA_KEY = "full_data"
+SUMMARY_KEY = "summary"
```

Comment:
would it be clearer to say `FULL_DATA_KEY` and `SUMMARY_KEY`? technically its all "data"

Reply:
thanks for updating this